### PR TITLE
#64 - Fix Navigation response

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

**Comment by edichoska** [comment]
@Mikivitkovski this looks much better.
Can you also include the logo?
I will send you the files.

**Comment by RevanchistX** [comment]
@Mikivitkovski  Please include ticket number in the pull request name

**Comment by RevanchistX** [comment]
Updated/Linked to #64 as it updates the Navigation. Maybe should be #68 or #69, Im not sure. If you have more context, update the ticket accordingly

File: components/reusable-components/navigation/navigation.module.scss [hunk 1]
```diff
@@ -27,34 +27,28 @@
   margin-left: 0.55em;
```
Comment (edichoska) [inline components/reusable-components/navigation/navigation.module.scss:27]: remove this margin, so the spacing from left and right of the navigation are the same

**Review by edichoska — APPROVED** [review]
🧷

**Pull request merged**